src/main/g8/default.properties

```diff
@@ -2,3 +2,4 @@ description = This template generates a Play Scala project
 name=play-scala-seed
 organization=com.example
 verbatim=*.css *.js *.png logback.xml
+play_version = maven(com.typesafe.play, sbt-plugin)
```

Comment:
We don't need to keep updating the version, we can let is be automatically fetched from maven central.

Reply:
Nice!